File: java/gugudan.java
import java.util.*;

public class gugudan {
	public static void main (String[] args) {
		Scanner s = new Scanner(System.in);
		do {  
			System.out.print("Insert number(0:Exit) : ");
			int i = s.nextInt();

			if (i == 0) {
				System.exit(1);
			} else if (i>1 && i<10) {
				for(int j=1; j<10; j++) {
					System.out.println( i +" * "+ j + " = " + i*j );

				}
			} else {
				System.out.println("Insert Number 2~9. Retype again~!!");
			}
		} while(true);
	}
}

File: gcc/gugudan.c
#include <stdio.h>

int  main() {
    while(1) {
        printf("Insert number(0:Exit) : ");
        int i, j;
        scanf("%d", &i);
        if(i == 0) {
            return 0;
        } else if (i>1 && i<10) {
            for(j=1; j<=9; j++)
                printf("%d * %d = %d \n", i, j, i*j);
        } else {
            printf("Insert number 2~9. Retype Again~!!\n");
        }
    }
}
